boards/esp32-ethernet-kit-v1_0/include/board.h

```diff
@@ -25,14 +25,6 @@
 extern "C" {
 #endif
 
-/**
- * @brief Initialize the board specific hardware
- */
-static inline void board_init(void) {
-    /* there is nothing special to initialize on this board */
-    board_init_common();
```

Comment:
`board_init_common` function should then be removed from `cpu/common/esp32/board_common.c`.

Reply:
It's still used by `esp32-ttgo-t-beam()`

Reply:
> It's still used by `esp32-ttgo-t-beam()`

@benpicco In fact, `board_init_common()` in `cpu/common/esp32/board_common.c` is just empty. May I remove it?

Reply:
Sure!